### DISCUSSION TITLE
Respawn

### DISCUSSION THREAD

**Discussion by korda2310** [Vdauphin/HeartsAndMinds#1361] [Question and answer]
Hello, how to use just the vacuum (respawn_west ) and not the ticket system and all that goes with it. Thanks in advance.

**Comment by Vdauphin** [comment]
Hello,

Have you check the mission parameters?

Cheers

**Reply by korda2310** [reply]
I can't find where I can remove all the system for tickets and also remove the random respawn. I would like to put only an empty respawn (respawn_west) on the mission. Sorry for my English via translator. (FR)

**Reply by Vdauphin** [reply]
Set default to `-1` here:
https://github.com/Vdauphin/HeartsAndMinds/blob/65c8999d174fc396b95260e5c3ddbb416e1093af/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L68-L72

> also remove the random respawn. 

What is the random respawn?

**Reply by korda2310** [reply]
Thanks, in fact I would like to respawn without this random system like on this picture.
![20220426183102_1](https://user-images.githubusercontent.com/104427562/165349226-8bb68614-c841-4753-ba98-9d0810ebc05c.jpg)

**Reply by Vdauphin** [reply]
Well just click on respawn and you will respawn at the base
It is not random

**Comment by Vill63** [comment]
Hello Vdauphin,
I hope you're doing fine !
I think I have the same request as Korda. Lets say the playable soldiers are set in pos A. Whenever I die I can select to respawn at the same pos A. But if I'd like to respawn where my "respawn-west" marker is set, lets say position B. When I die, I only have option A on which I can respawn.
How can I make the vanilla respawn marker work again ?

Cheers,
Vill

**Reply by STyx2909** [reply]
It should work in any case to be proposed to respawn at FOB.

**Reply by Vdauphin** [reply]
Hello, good here :)

It is like that because when your base is on a ship, "respawn-west" marker let you spawn in the sea...

comment this line here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/init_player.sqf#L16

Cheers

**Reply by Vill63** [reply]
Makes sense, you’re the best !
Thank you Sir !
